s5a/__init__.py

```diff
@@ -25,10 +27,29 @@ def load_ncfile(ncfile):
     :rtype: pandas.core.frame.DataFrame
     """
 
+    # If no data variable name is given, choose one based
+    # on the file name from the following lookup table of
+    # product string identifiers with their corresponding
+    # default variable names.
+    prod_var_refs = {'L2__CO_____': 'carbonmonoxide_total_column',
+                     'L2__O3_____': 'ozone_total_vertical_column',
+                     'L2__SO2____': 'sulfurdioxide_total_vertical_column',
+                     'L2__CH4____': 'methane_mixing_ratio_bias_corrected',
+                     'L2__HCHO___': 'formaldehyde_tropospheric_vertical_column',  # noqa: E501
+                     'L2__NO2____': 'nitrogendioxide_tropospheric_column'
+                     }
```

Comment:
You could write

```python
    prod_var_refs = {
        'L2__CO_____': 'carbonmonoxide_total_column',
        'L2__O3_____': 'ozone_total_vertical_column',
        'L2__SO2____': 'sulfurdioxide_total_vertical_column',
        'L2__CH4____': 'methane_mixing_ratio_bias_corrected',
        'L2__HCHO___': 'formaldehyde_tropospheric_vertical_column',
        'L2__NO2____': 'nitrogendioxide_tropospheric_column',
    }
```

So you do not need to disable the linter.

Reply:
We could also simply check if variables exist instead of relying on the file name.
Something like:

```python
variable_names = ('carbonmonoxide_total_column', 'ozone_total_vertical_column', …)

for variable_name in variable_names:
    if hasattr(variables, variable_name):
        data_variable_name = variable_name
        break
```

Reply:
I like @lkiesow's idea. It is much less error prone. I personally often rename files for testing. We have already done this for the testing in the CI.

Reply:
Yep, good idea. [Done](https://github.com/emissions-api/sentinel5algorithms/pull/25/commits/f977464694935c9d95bb1f113bb9af8425325867).